Source/WebCore/Modules/compression/DecompressionStream.js

```diff
@@ -27,12 +27,12 @@ function initializeDecompressionStream(format)
 {
     "use strict";
 
-    const errorMessage = "DecompressionStream requires a single argument with the value 'deflate', 'deflate-raw', or 'gzip'.";
+    const errorMessage = "DecompressionStream requires a single argument with the value 'brotli', 'deflate', 'deflate-raw', or 'gzip'.";
 
     if (arguments.length < 1)
         @throwTypeError(errorMessage);
 
-    const algorithms = ['gzip', 'deflate', 'deflate-raw'];
+    const algorithms = ['brotli', 'gzip', 'deflate', 'deflate-raw'];
```

Comment:
We probably only want to include brotli in the list of algorithms on platforms that support it.

Reply:
A couple thoughts here.

1. Do we have a way to perform platform checks in JS (similar to `#if PLATFORM`)? 
2. We could move algorithms support checking to the C++ code where we can utilize the platform macros.

Reply:
2

Reply:
I took care of this another way, by returning an exception on non-Cocoa platforms when trying to create a brotli compressor or decompressor.